fleetshard/pkg/k8s/secret.go

```diff
@@ -0,0 +1,69 @@
+package k8s
+
+import (
+	"context"
+	"fmt"
+	"sort"
+
+	"github.com/pkg/errors"
+	corev1 "k8s.io/api/core/v1"
+	apiErrors "k8s.io/apimachinery/pkg/api/errors"
+	ctrlClient "sigs.k8s.io/controller-runtime/pkg/client"
+)
+
+const (
+	centralTLSSecretName        = "central-tls"         // pragma: allowlist secret
+	centralDBPasswordSecretName = "central-db-password" // pragma: allowlist secret
+)
+
+var secretsToWatch = []string{
+	centralTLSSecretName,
+	centralDBPasswordSecretName,
+}
+
+// SecretService is responsible for reading and writing secrets
+// This servise is specific to ACS Managed Services and provides methods work on specific secrets
```

Comment:
```suggestion
// SecretService is responsible with reading specified secrets from a Kubernetes namespace
```

There's no writing as of yet, I guess you plan to add it in the future, but for now it's misleading. Also the name SecretService sounds a bit funny, maybe we can come up with another name?